pkg/parse/parse.go

```diff
@@ -266,15 +268,17 @@ func checkCalls(mod *sysl.Module, appname string, epname string, dst *sysl.State
 	case *sysl.Statement_Call:
 		app := syslutil.GetApp(s.Call.Target, mod)
 		if app == nil {
-			logrus.Warnf("%s::%s calls non-existent App: %s",
-				appname, epname, s.Call.Target.Part)
+			//TODO: delete this? this removes non-existent app call lint
```

Comment:
Can this be removed now that we handle linting of non existent apps elsewhere?
Same for the commented out code below